[2021-07-16 16:29:38,090] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:28:37.330568+00:00 [queued]>
[2021-07-16 16:29:38,130] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:28:37.330568+00:00 [queued]>
[2021-07-16 16:29:38,131] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 16:29:38,132] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-07-16 16:29:38,133] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 16:29:38,157] {taskinstance.py:1087} INFO - Executing <Task(TriggerDagRunOperator): restart_test> on 2021-07-16T16:28:37.330568+00:00
[2021-07-16 16:29:38,183] {standard_task_runner.py:52} INFO - Started process 7990 to run task
[2021-07-16 16:29:38,210] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'psotgrressdftest', 'restart_test', '2021-07-16T16:28:37.330568+00:00', '--job-id', '1081', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/POSTGRESRSERS.py', '--cfg-path', '/tmp/tmpc2c243gj', '--error-file', '/tmp/tmpbstjd98s']
[2021-07-16 16:29:38,214] {standard_task_runner.py:77} INFO - Job 1081: Subtask restart_test
[2021-07-16 16:29:38,334] {logging_mixin.py:104} INFO - Running <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:28:37.330568+00:00 [running]> on host 13346afafa5b
[2021-07-16 16:29:38,459] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=psotgrressdftest
AIRFLOW_CTX_TASK_ID=restart_test
AIRFLOW_CTX_EXECUTION_DATE=2021-07-16T16:28:37.330568+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-16T16:28:34.096604+00:00
[2021-07-16 16:29:43,985] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=psotgrressdftest, task_id=restart_test, execution_date=20210716T162837, start_date=20210716T162938, end_date=20210716T162943
[2021-07-16 16:29:44,054] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-16 16:29:44,166] {local_task_job.py:151} INFO - Task exited with return code 0
